version 4/03 environments/04 Production Configuration/README.md

```diff
@@ -0,0 +1,341 @@
+# 04 Production Configuration
+
+In this demo we are going to create different builds for each environment.
+We will learn how to configure it and how to reduce bundle file sizes.
+
+We will start from sample _03 Environments/03 HMR_.
+
+Summary steps:
+- Add base webpack config file
+- Add webpack-merge package.
+- Add development config file.
+- Add production config file.
+
+# Steps to build it
+
+## Prerequisites
+
+Prerequisites, you will need to have nodejs installed in your computer. If you want to follow this step guides you will need to take as starting point sample _03 hmr_.
+
+## steps
+
+- `npm install` to install previous sample packages:
+
+```
+npm install
+```
+
+- We are going to start with creating a base webpack configuration file where we are going to add the common features for both environments (`dev` and `prod`). One of the feature which we are split between environments is `sourcemaps`. [Webpack recommends](https://webpack.js.org/guides/production/#source-mapping) that we enable source maps for `production` environment, due to are useful for debugging and to run benchmark tests, but we have to disable it for normal use.
```

Comment:
Fixing some grammar / wording:

- We are will start by creating base webpack configuration that will hold all the common feature for both environments (_dev_ and _prod). Then we will place specific configuration for dev and production in their config files (e.g. sourcemaps). Webpack, recommends enabling sources maps for development environment (debuggging purposes), but disable it for normal use.

Reply:
done